pgraph/pgraph.go

```diff
@@ -52,12 +53,13 @@ type Flags struct {
 // * IOW, you might see package -> file -> service (where package runs first)
 // * This is also the direction that the notify should happen in...
 type Graph struct {
-	Name      string
-	Adjacency map[*Vertex]map[*Vertex]*Edge // *Vertex -> *Vertex (edge)
-	Flags     Flags
-	state     graphState
-	mutex     *sync.Mutex // used when modifying graph State variable
-	wg        *sync.WaitGroup
+	Name       string
+	Adjacency  map[*Vertex]map[*Vertex]*Edge // *Vertex -> *Vertex (edge)
+	Flags      Flags
+	prometheus *prometheus.Prometheus // the prometheus instance
```

Comment:
nit: Can you put this at the end with one newline so that it sits by itself please?